frontend/src/atoms/__snapshots__/MD2Tabs.test.tsx.snap

```diff
@@ -0,0 +1,351 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Input gracefully handles an out of bound selectedTab value 1`] = `
```

Comment:
What is this file @kwasi ?

Reply:
This repo is using Jest snapshots: https://jestjs.io/docs/en/snapshot-testing

This is a new snapshot for the MD2Tabs component.

https://github.com/kubeflow/metadata/tree/master/frontend/src/atoms/__snapshots__